packages/test/test-end-to-end-tests/src/test/summarizeRestart.spec.ts

```diff
@@ -0,0 +1,203 @@
+			{
+				eventName: "fluid:telemetry:Summarizer:Running:HandleLastSummaryAckError",
+				error: "Restarting summarizer instead of refreshing",
+			},
+		],
+		async () => {
+			const container = await createContainer();
+			const { container: summarizingContainer, summarizer } = await createSummarizer(
+				provider,
+				container,
+				undefined,
+				undefined,
+				mockConfigProvider(settings),
+			);
+
+			const { container: summarizingContainer2, summarizer: summarizer2 } =
+				await createSummarizer(
+					provider,
+					container,
+					undefined,
+					undefined,
+					mockConfigProvider(settings),
+				);
+
+			await summarizeNow(summarizer);
+
+			const dataStore = await requestFluidObject<ITestDataObject>(container, "default");
+			dataStore._root.set("an", "op");
+
+			await provider.ensureSynchronized();
+			const container2 = summarizingContainer2 as any;
+			// Intentional hack to allow two summaries to be submitted at the same time
+
+			const summarizeResults = summarizer.summarizeOnDemand({ reason: "end-to-end test" });
+			container2._clientId = (summarizer2 as any).runtime.summarizerClientId;
+
+			const summarizeResults2 = summarizer2.summarizeOnDemand({ reason: "end-to-end test" });
+
+			await Promise.all([
+				summarizeResults.summarySubmitted,
+				summarizeResults2.summarySubmitted,
+			]);
+
+			await Promise.all([
+				summarizeResults.summaryOpBroadcasted,
+				summarizeResults2.summaryOpBroadcasted,
+			]);
+
+			await Promise.all([
+				summarizeResults.receivedSummaryAckOrNack,
+				summarizeResults2.receivedSummaryAckOrNack,
+			]);
+			const ack = await summarizeResults.receivedSummaryAckOrNack;
+			const nack = await summarizeResults2.receivedSummaryAckOrNack;
+
+			await provider.ensureSynchronized();
+
+			assert(ack.success, "Should be an ack");
+			assert(!nack.success, "Should be a nack");
+			assert(summarizingContainer2.closed, "Unknown acks should close the summarizer");
+			assert(!summarizingContainer.closed, "summarizer1 should not be closed");
+			assert(!container.closed, "Original container should not be closed");
+		},
+	);
+
+	itExpects(
+		"Closes the summarizing client instead of refreshing when loading from an older summary",
+		[
+			{
+				eventName: "fluid:telemetry:Container:ContainerClose",
+				error: "Restarting summarizer instead of refreshing",
+			},
+			{
+				eventName: "fluid:telemetry:SummarizerNode:refreshLatestSummary_cancel",
+				error: "Restarting summarizer instead of refreshing",
+			},
+			{
+				eventName: "fluid:telemetry:Summarizer:Running:HandleLastSummaryAckError",
+				error: "Restarting summarizer instead of refreshing",
+			},
+		],
+		async () => {
+			const container = await createContainer();
+			const { container: summarizingContainer, summarizer } = await createSummarizer(
+				provider,
+				container,
+				undefined,
+				undefined,
+				mockConfigProvider(settings),
+			);
+
+			// s1
+			const { summaryVersion } = await summarizeNow(summarizer);
```

Comment:
`summaryVersion1` to make it explicit that this is from summary1. That way when you load a summarizer from this, its clear.